src/app/containers/Image/index.jsx

```diff
@@ -51,11 +55,20 @@ const ImageContainer = ({ blocks }) => {
   const ratio = (height / width) * 100;
   const rawImageSrc = getRawImageSrc(originCode, locator);
 
+  let Wrapper = GridItemConstrainedLargeNoMargin;
+
+  if (height === width) {
```

Comment:
Double-checking with product whether we want an exact "this is a perfectly square image" (500 x 500) or "this is what the human eye would think is square" (500 x 495) for this line. 

Update: @jimjohnsonrollings confirmed "exactly" square for simplicity of the CMS guidelines